packages/components/pie-divider/src/index.ts

```diff
@@ -24,20 +26,37 @@ export class PieDivider extends LitElement implements DividerProps {
     @validPropertyValues(componentSelector, orientations, defaultProps.orientation)
     public orientation = defaultProps.orientation;
 
+    @property({ type: String })
+    public label = defaultProps.label;
+
     render () {
-        const { variant, orientation } = this;
+        const { variant, orientation, label } = this;
 
         const classes = {
-            'c-divider': true,
             'c-divider--inverse': variant === 'inverse',
             'c-divider--vertical': orientation === 'vertical',
+            'c-divider--labelled': label.length > 0 && orientation === 'horizontal',
         };
 
         return html`
-            <hr
-                data-test-id="pie-divider"
-                aria-hidden="true"
-                class="${classMap(classes)}" />`;
+            ${label && orientation === 'horizontal'
+            ? html`
+                <div
+                    id="${componentSelector}"
+                    data-test-id="${componentSelector}"
+                    class="c-divider${classMap(classes)}"
```

Comment:
Just for simplicity I would probably just add 'c-divider: true` to the class object above

Reply:
also much easier to read - perhaps we should document this as a proposed convention @xander-marjoram?